packages/lib-classifier/src/store/utils/session.js

```diff
@@ -24,18 +17,7 @@ const sessionUtils = {
 
   getSessionID () {
     const stored = (storage.getItem('session_id')) ? JSON.parse(storage.getItem('session_id')) : this.generateSessionID()
-    let { id, ttl } = stored
-
-    if (ttl < Date.now()) {
```

Comment:
This condition was comparing a Date() string to a Date.now() number and therefore never met. It was copied directly from PFE's codebase, so I'll be doing the same cleanup over there.